modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/jdbc/JdbcQueryTest.java

```diff
@@ -108,6 +110,67 @@ public void testSimpleQuery() throws Exception {
         stmt.close();
     }
 
+    /** Test batched execution of statement. */
+    @Test
+    public void testBatch() throws Exception {
+        stmt.execute("CREATE TABLE Person(\"id\" INT, PRIMARY KEY(\"id\"), \"name\" VARCHAR)");
+
+        grid(0).context().cache().context().exchange().affinityReadyFuture(
```

Comment:
Looks confusing and redundant (at least while partition awareness is disabled by default for JDBC). I know, it was copied from existing test, but perhaps we should fix this existing test too.